### ISSUE TITLE
JSTestDriver plugin for Webstorm does not appear to pick up changes to SUT files and requires JSTestDriver Server restart for changes to take effect.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#172]
```
What steps will reproduce the problem?
1. Start the server and capture a browser
2. Given some spec that already exists, modify one of classes under test to 
fail that test.
3. Run JSTestDriver configuration

What is the expected output? What do you see instead?
The test should fail, but it succeeds.

What version of the product are you using? On what operating system?
I am using WebStorm 2.0 EAP build 98.364 on Mac OSX 10.6.5 with plugin v.1.2.2a

Please provide any additional information below.
This is NOT an intermittent issue.  It happens every single time.  Any changes 
I make to actual tests get picked up without fail, but that is not the case 
when changes are made to the SUT code.  It appears as though SUT file are being 
cached. Makes TDD a pain, having to restart the server constantly.  Definitely 
an issue to those not aware of the problem, as was the case for me, as any 
passed tests during a dev session, may no longer pass the next day, or next dev 
session when WebStorm and JSTestDriver server are restarted.

```

Original issue reported on code.google.com by `ireney.b...@gmail.com` on 6 Dec 2010 at 10:19

**Comment by GoogleCodeExporter** [comment]
Original comment by `aeagle22206` on 11 Apr 2011 at 7:29
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
If this bug is still reproducible in the plugin v1.3 or higher, could you 
please provide test sources (*.js and config file)?
```

Original comment by `Sergey.S...@gmail.com` on 10 May 2011 at 10:28

**Comment by GoogleCodeExporter** [comment]
```
This bug has been successfully reproduced in WebStorm 2.0 EAP build 98.364, Mac 
OSX 10.6.3, JsTestDriver IDEA plugin v1.2.2a.
It is fixed in WebStorm 2.1.1, Mac OSX 10.6.3, JsTestDriver IDEA plugin v1.3.5.
```

Original comment by `Sergey.S...@gmail.com` on 28 Jun 2011 at 6:36
- Changed state: **Fixed**
